src/js/calculator.js

```diff
@@ -0,0 +1,110 @@
+export class Calculator {
+  #firstNumberString;
+  #secondNumberString;
+  #operator;
+  #displayText;
+  #hasClickedOperator;
+
+  constructor() {
+    this.#firstNumberString = "";
+    this.#secondNumberString = "";
+    this.#operator = "";
+    this.#displayText = "";
+    this.#hasClickedOperator = false;
+  }
+
+  #updateDisplayText = (text) => {
+    const total = document.querySelector("#total");
+    total.innerText = text;
+  };
+
+  #initializeInputData = () => {
+    this.#firstNumberString = "";
+    this.#secondNumberString = "";
+    this.#operator = "";
+    this.#hasClickedOperator = false;
+  };
+
+  #isLessThanThreeDigits = (numberString) => {
+    if (numberString.length < 3) {
+      return true;
+    }
+    return false;
+  };
```

Comment:
비교연산을 바로 반환해보면 좋을 것 같아요
```suggestion
  #isLessThanThreeDigits = (numberString) => numberString.length < 3
```